pkg/netstate/netstate.go

```diff
@@ -83,6 +107,17 @@ func (s *Server) List(ctx context.Context, req *pb.ListRequest) (*pb.ListRespons
 	s.logger.Debug("entering netstate list")
 
 	pathKeys, err := s.DB.List()
+
+	xAPIKeyBytes := []byte(req.XApiKey)
+	if err := validateAuth(xAPIKeyBytes); err != nil {
+		c := []byte("Unauthorized Request")
+		s.logger.Error("unauthorized request")
+
+		return &pb.ListResponse{
+			Paths: [][]byte{c},
```

Comment:
nit-- 
```Go
return &pb.ListResponse{
    Paths: [][]byte{
        []byte("Unauthorized Request"),
    },
}, err
```

Reply:
thanks! totally memoed; i ended up deleting that part.